### PR TITLE
Do not use spread operator to concatenate large arrays (Closes #153)

### LINKED ISSUE #153: [Bug] Maximum call stack size exceeded
When I pass in an large set  of 'training data' as input I get `Maximum call stack size exceeded

Here's my code
```
// load all .txt files from ../assets/
const files = fs.readdirSync('./assets/').filter(file => file.endsWith('.txt'));
const data = [];

// open all of the files and read them into a single string
for (let i = 0; i < files.length; i++) {
    const file = files[i];
    const fileContents = fs.readFileSync(`./assets/${file}`, 'utf-8');
    data.push(fileContents);
}
```

Calling the Data  with
```
let response = await generator(text + prompt + "\n\nResponse: ",
        {
            max_new_tokens: 1200,
            temperature: 0.5,
            top_k: 50,
            top_p: 0.3,
            repetition_penalty: 2.3,
            num_return_sequences: 1
        }
    )
```

Here's the response from the logs
```
RangeError: Maximum call stack size exceeded
    at Function.post_process (file:///home/sister/repos/bots/glaza/node_modules/@xenova/transformers/src/tokenizers.js:1236:30)
    at Function._call (file:///home/sister/repos/bots/glaza/node_modules/@xenova/transformers/src/tokenizers.js:1162:21)
    at Function.closure [as post_processor] (file:///home/sister/repos/bots/glaza/node_modules/@xenova/transformers/src/utils/core.js:62:28)
    at Function.encode (file:///home/sister/repos/bots/glaza/node_modules/@xenova/transformers/src/tokenizers.js:2115:35)
    at file:///home/sister/repos/bots/glaza/node_modules/@xenova/transformers/src/tokenizers.js:1946:45
    at Array.map (<anonymous>)
    at Function._call (file:///home/sister/repos/bots/glaza/node_modules/@xenova/transformers/src/tokenizers.js:1946:31)
    at Function.closure [as tokenizer] (file:///home/sister/repos/bots/glaza/node_modules/@xenova/transformers/src/utils/core.js:62:28)
    at Function._call (file:///home/sister/repos/bots/glaza/node_modules/@xenova/transformers/src/pipelines.js:400:30)
    at closure (file:///home/sister/repos/bots/glaza/node_modules/@xenova/transformers/src/utils/core.js:62:28)
```

Running
```
Linux 6.3.4
Node 20.2.0
```

I'm sure there could be an error in my code, implementation or logic.. but if this is _not_ wrong, I would be happy to help resolve the bug or any issues this may have exposed.

### PR DESCRIPTION
**Pull request by xenova**
Fixes "Maximum call stack size exceeded" which is caused by using the spread operator to concatenate large arrays.

### CODE REVIEW

File: tests/tokenizers.test.js [hunk 1]
```diff
@@ -33,3 +33,13 @@ describe('Tokenizers', () => {
         });
     }
 });
+
+describe('Edge cases', () => {
+    it('should not crash when encoding a very long string', async () => {
+        let tokenizer = await AutoTokenizer.from_pretrained('t5-small');
+
+        let text = String.prototype.repeat.call('Hello world! ', 50000);
```
Comment (kungfooman) [inline tests/tokenizers.test.js:41]: `'Hello world! '.repeat(50000);` :pinched_fingers: 

Kinda sad to see the nice array spreading syntax gone, but this is the reality of large arrays, nice work :1st_place_medal:
